<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Tobias Junghans</author>
 <class>FileTransferDialog</class>
 <widget class="QDialog" name="FileTransferDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File transfer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="filetransfer.qrc">
    <normaloff>:/filetransfer/applications-office.png</normaloff>:/filetransfer/applications-office.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="overwriteExistingFiles">
        <property name="text">
         <string>Overwrite existing files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="transferOnly">
        <property name="text">
         <string>Transfer only</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="transferAndOpenProgram">
        <property name="text">
         <string>Transfer and open file(s) with associated program</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="transferAndOpenFolder">
        <property name="text">
         <string>Transfer and open destination folder</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="fileListView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDestination">
     <property name="text">
      <string>Destination folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leDestination">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>This can be set by an admin using Veyon Configurator</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>overwriteExistingFiles</tabstop>
  <tabstop>transferOnly</tabstop>
  <tabstop>transferAndOpenProgram</tabstop>
  <tabstop>transferAndOpenFolder</tabstop>
  <tabstop>fileListView</tabstop>
 </tabstops>
 <resources>
  <include location="filetransfer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FileTransferDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FileTransferDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
